### PR TITLE
Sam fiftyone

### PR DESCRIPTION
**Pull request by jacobmarks**
- As of FiftyOne 0.21, the ubuntu issue should be resolved, so I've removed that section
- I've refactored the last section on segmenting an entire dataset - there was a subtle bug there - but now it works!

### CODE REVIEW

**Comment by ryanloney** [comment]
It crashes here on my Windows 11, Python 3.10.11 environment: 
image

File: notebooks/237-segment-anything/README.md [hunk 1]
```diff
@@ -8,20 +8,14 @@ The [Segment Anything Model (SAM)](https://github.com/facebookresearch/segment-a
...
 
 Segment Anything Model is a generalization of these two classes of approaches. It is a single model that can easily perform both interactive segmentation and automatic segmentation.
-This notebook shows an example of how to convert and use Segment Anything Model in OpenVINO format, allowing it to run on a variety of platforms that support an OpenVINO.
+The two notebooks in this folder show how to convert and use the Segment Anything Model in OpenVINO format, allowing it to run on a variety of platforms that support OpenVINO.
 
-The notebook demonstrates how to work with model in 2 modes:
+The first notebook, `237-segment-anything.ipynb`, demonstrates how to work with model in tnteractive segmentation mode and automatic segmentation mode.
```
Comment (itrushkin) [inline notebooks/237-segment-anything/README.md:13]:
``` suggestion
The first notebook, `237-segment-anything.ipynb`, demonstrates how to work with model in interactive segmentation mode and automatic segmentation mode.

**Review by itrushkin — CHANGES_REQUESTED** [review]

File: notebooks/237-segment-anything/237-segment-anything-fiftyone.ipynb [hunk 2]
```diff
@@ -0,0 +1,2331 @@
+{
```
Comment (itrushkin) [inline notebooks/237-segment-anything/237-segment-anything-fiftyone.ipynb:1]: <p>Empty cell</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/openvinotoolkit/openvino_notebooks/pull/1138/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='19'/>

Comment (itrushkin) [inline notebooks/237-segment-anything/237-segment-anything-fiftyone.ipynb:1]: <pre class="ql-syntax" spellcheck="false">Line #3.    import fiftyone as fo
</pre><p>On Ubuntu 22.04, this import fails due to https://github.com/voxel51/fiftyone/issues/1803.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/openvinotoolkit/openvino_notebooks/pull/1138/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='20'/>

**Comment by andrei-kochin** [comment]
Closing this one due to no activity

**Pull request closed without merging**